dallinger/command_line/docker_ssh.py

```diff
@@ -325,18 +335,32 @@ def deploy(mode, server, dns_host, config_options):  # pragma: no cover
     grant_roles_script = (
         f'grant all privileges on database "{experiment_id}" to "{experiment_id}"'
     )
+
+    if archive_path is not None:
+        print(f"Loading database data from {archive_path}")
```

Comment:
Should this be `click.echo()` or something? I see print() is used extensively here already, so never mind.